controller/src/test/java/io/pravega/controller/server/v1/ControllerServiceTest.java

```diff
@@ -67,20 +67,21 @@
 
     private final StreamMetadataStore streamStore = StreamStoreFactory.createInMemoryStore(executor);
 
-    private final StreamMetadataTasks streamMetadataTasks;
-    private final StreamTransactionMetadataTasks streamTransactionMetadataTasks;
-    private final ConnectionFactoryImpl connectionFactory;
-    private final ControllerService consumer;
+    private StreamMetadataTasks streamMetadataTasks;
```

Comment:
Why is final removed from all of these? I don't see them modified elsewhere.

Reply:
actually they were  being initialized in a constructor in these test.. 
now i have changed it to be initialized in `@Before setUp` method and closed in `@After tearDown` which is called before and after each test. since its not initialized in constructor, rather no test constructor, so no final business..